include/seqan3/range/decorator/gap_decorator.hpp

```diff
@@ -1038,6 +655,390 @@ template <std::ranges::view urng_t>
 gap_decorator(urng_t range) -> gap_decorator<urng_t>;
 //!\}
 
+/*!\brief A special iterator that enforces std::random_access_iterator albeit its complexity is logarithmic.
```

Comment:
```suggestion
/*!\brief A special iterator that models std::random_access_iterator albeit its complexity is logarithmic.
```

Reply:
I think using the word enforce makes it more clear to the user, that something unusual is happening. Concrete that it models only syntactically the RA iterator. At least that was my intention. Do you disagree?

Reply:
ok fine